opm/simulators/wells/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -1060,6 +1068,12 @@ namespace Opm
             for (const auto& x : injection_group_vrep_rates) {
                 data[pos++] = x.second;
             }
+            for (const auto& x : current_alq_) {
+                data[pos++] = x.second;
+            }
+            // for (const auto& x : default_alq_) {
+            //     data[pos++] = x.second;
+            // }
             assert(pos == sz);
```

Comment:
Does this mean we only need the default alq value locally?

Reply:
I think the design of the default/current alq system is confusing, and hope we can find something better. As it is, I struggled to find the bug here since it happened in a const function that modified the state (the alq members are mutable precisely to allow this). What is the behaviour the default_alq_ is intended to capture?

Reply:
> Does this mean we only need the default alq value locally?

I think the answer is yes.

(Did not notice I had not answered your question).